versions_to_build

```diff
@@ -1,2 +1,2 @@
-1.2.7-0.1.3
-1.3.5-0.0.4
+1.2.7-0.1.4
+1.3.5-0.0.5
```

Comment:
@angelofenoglio we've discussed yesterday with @juanmatias that since both versions of the Dockerfile install the exact same versions of all the components for both based terraform images `1.2.7` and `1.3.5` it makes more sense that the release version reflects this with the same version.

```
1.2.7-0.1.4
1.3.5-0.1.4
```
This will also simplify the release management automation, eg: being able to then add the `1.2.7-0.1.4-debian` version tag. And in case there is really a difference in between the installed components extending the base tf images 1.2.7 and 1.3.5, we can reflect this using different patch or minor vers.

CC: @diego-ojeda-binbash

Reply:
Seems reasonable to me to, I'll quickly patch it!